OpenTrons/opentrons-api
# refactor(app): remove bouncing logo screen

# Overview
Remove Sleep screen(bouncing logo screen) and turn off the backlight when the counter hits the sleep setting.
This branch adds the empty screen to keep using the same method as well as the bouncing logo screen.
☝️ I got this idea from Brian 😄 

Currently the Touchscreen sleep layout is broken but this will be solve by https://github.com/Opentrons/opentrons/pull/12650



# Test Plan
- Settings -> Touchscreen sleep -> select a sleep time
- Wait for x minutes
- Check the backlight is turned off (black screen)
- tap the screen and the app shows the screen you saw the before sleeping


# Changelog
- update OnDeviceDisplayApp
- remove Sleepscreen


# Review requests



# Risk assessment
low
